# HG changeset patch
# User Rik <rik@octave.org>
# Date 1523559937 25200
#      Thu Apr 12 12:05:37 2018 -0700
# Node ID 920adb4051a3ba26e1c24d46860ecc85c1e25582
# Parent  581d3a13d7e27b5fcbd62f8e6ddad1d6a02bca12
doc: Update manual title page to edition 5.

* octave.texi: Update Edition to 5.

diff --git a/doc/interpreter/octave.texi b/doc/interpreter/octave.texi
--- a/doc/interpreter/octave.texi
+++ b/doc/interpreter/octave.texi
@@ -92,17 +92,17 @@ modified versions.
 @titlepage
 @title GNU Octave
 @subtitle A high-level interactive language for numerical computations
 @c FIXME: Would like to use @value{EDITION} but this variable is the
 @c        same as VERSION and contains MAJOR.MINOR.PATCH.  The variable
 @c        is automatically generated by Automake in version-octave.texi.
 @c        Need to use sed to strip off MINOR.PATCH numbers and place
 @c        the results in a new file, and then @include that new file.
-@subtitle Edition 4 for Octave version @value{VERSION}
+@subtitle Edition 5 for Octave version @value{VERSION}
 @subtitle @value{UPDATED-MONTH}
 @sp 2
 @multitable @columnfractions 0.4 0.025 0.65
 @item
 @flushright @image{octave_logo,2.1in}
 @end flushright
 @tab
 @c this is a spacer column
